contracts/token/ERC721/ERC721SingleRentalAgreement.sol

```diff
@@ -6,33 +6,30 @@ import "../../utils/introspection/ERC165.sol";
 
 /// @title ERC721 simple rental agreement.
 /// Define a simple rental agreement following the principles:
-///   - The rent is valid for a rental period
-///   - The rent has an expration date: after this date it cannot be started.
+///   - The rent is valid for a rental period. The rental period has to be over to finish the rental.
```

Comment:
The rent is valid for a rental period -> The rental is valid for a fixed period.

Reply:
changed it